torch/fx/experimental/proxy_tensor.py

```diff
@@ -590,6 +591,35 @@ def unwrap_proxy(self, e):
         else:
             return e
 
+    def create_node(self, *args, **kwargs):
+        '''
+        Add stack_trace metadata here if not already present,
+        by filtering out forward() stack frames
+        '''
+        node = super().create_node(*args, **kwargs)
+        if 'stack_trace' not in node.meta and node.op not in ["placeholder", "output"]:
+            user_frame = self._find_user_frame()
+            if user_frame:
+                summary = traceback.extract_stack(user_frame)
+                # we retain frames from forward() calls, or ops
+                # located in torch/__init__.py (e.g. sym_int, sym_constrain_range, vmap)
+                stack_trace = [frame for frame in summary if (
+                    frame.name == 'forward'
+                    or frame.filename.endswith('torch/__init__.py')
+                )]
+                # filter out forward() frames from fx/_symbolic_trace.py, export/_trace.py
+                # this is hardcoded, but leads to a much cleaner stack trace
+                stack_trace = [
+                    frame for frame in stack_trace if not (
+                        frame.filename.endswith('fx/_symbolic_trace.py')
```

Comment:
I'm not super familiar with what `_find_user_frame` does, but it seems like it already filters out `fx/_symbolic_trace.py` [here](https://fburl.com/code/ytmmswlk)? 

Separately -- maybe we could rewrite this filtering to match the code in `_find_user_frame`, where the files we want to filter are in a list rather than having multiple `frame.filename.endswith` calls

Reply:
Thank you for the comments! Just addressed these.

For `_find_user_frame` I think it only filters for the one returned frame, but the stack might still contain those files